torchaudio/transforms/_transforms.py

```diff
@@ -355,6 +355,13 @@ class MelScale(torch.nn.Module):
             (area normalization). (Default: ``None``)
         mel_scale (str, optional): Scale to use: ``htk`` or ``slaney``. (Default: ``htk``)
 
+    Example
+        >>> waveform, sample_rate = torchaudio.load('test.wav', normalize=True)
```

Comment:
The previous codes use single quotation marks for `test.wav`, they should be changed to double quotation marks to fit linter check. Let's also use double quotes in this PR.

Reply:
Sounds good! I'm happy to do a sweep through other examples to change to double quotes. If there are any other code consistency thoughts you have, please let me know and I will incorporate them as well.

Reply:
Changed to double quotes. ...will plan to use a separate PR to change double quotes for other transform examples.

Reply:
I made a draft PR for changing double quotes https://github.com/pytorch/audio/pull/2618. That should fix the ones in functional/transform. Feel free to open a PR for other modules.

Reply:
Ahh got it, thanks for the heads up!